README.md

```diff
@@ -126,7 +138,58 @@ const sendBundleResponse = await skipBundleClient.sendBundle(signedBundle, DESIR
 ```
 
 `DESIRED_HEIGHT_FOR_BUNDLE` should be a number, where 0 asks the Sentinel to autodetermine the next height.
+### Terra and others using terra.js / feather.js
+Terra and [alliance](https://github.com/terra-money/alliance) chains usually use an alternative client library called [terra.js](https://github.com/terra-money/terra.js), it was rebranded to [feather.js](https://github.com/terra-money/feather.js) in late 2022 to add multichain support (i.e. similar to cosmjs, you can use it to interact with not only terra but also other cosmos SDK chains, most likely alliance chains that share security with Terra).
+
+The overall logic is similar to using cosmjs. Modified from the [send native token example](https://docs.terra.money/develop/feather-js/common-examples#sending-native-tokens).
+
+Import skip and feather.
+```
+import { SkipBundleClient } from '@skip-mev/skipjs'
+import { LCDClient, MnemonicKey, MsgSend } from '@terra-money/feather.js'
+```
 
+Create LCD and get mnemonic key.
+```
+const lcd = new LCDClient({
+    URL: LCD_ENDPOINT,
+    chainID: CHAIN_ID,
+})
+
+const mk = new MnemonicKey({
+  mnemonic:'', // Insert your mnemonic
+})
+const wallet = lcd.wallet(mk)
+const privKey = mk.privateKey
+```
+
+Construct message of send token to your contact address and another send token to [skip auction house address](https://docs.skip.money/searcher#winning-the-auction).
```

Comment:
The bundle doesn't need to include a send to a contract specifically, so let's not make this state anything about contract addresses.

Reply:
Are you suggesting I remove the send to my contact or send to auction house?

Reply:
This example is following terra doc to send token to someone (contact, not contract)

Reply:
I rephrased it a little, let me know if it makes sense or what you suggest.